colossalai/auto_parallel/passes/runtime_apply_pass.py

```diff
@@ -45,6 +45,26 @@ def runtime_apply_for_iterable_object(node: Node, origin_dict: Dict, input_dict:
     return rst
 
 
+def get_mem_cost(node: Node, origin_dict: Dict, input_dict: Dict, node_index: int, user_node_index: int):
```

Comment:
This function is used to get real memory cost from `ShapeConsistencyManager.mem_cost()` as the `ShapeConsistencyManager` itself doesn't have the access to tensor type.